pallets/grants/src/lib.rs

```diff
@@ -214,6 +214,48 @@ pub mod pallet {
 
 			Ok(().into())
 		}
+
+		/// Initiate a bridge transfer of all vested funds to the given `eth_address` on the given `chain_id`.
+		/// This process will need to be completed by the bridge oracles.
+		#[pallet::call_index(4)]
+		#[pallet::weight(T::WeightInfo::bridge_all_vesting_schedules())]
+		pub fn bridge_all_vesting_schedules(
+			origin: OriginFor<T>,
+			eth_address: [u8; 20],
+			chain_id: u64,
```

Comment:
Shall we have something similar to the ink! contract where we verify the address' checksum & chain id instead of adding it to the event at the bottom?

Reply:
We would need to have it in the event anyway. But I agree if we can verify it we should do it as well. I will add it soon

Reply:
Actually the verify became sort of useless on the side of the smart contract for 2 reasons:
1. Our own Nodle Client and App convert any address to a valid address as recognized by the smart contract.
2. ZkSync has not opted https://eips.ethereum.org/EIPS/eip-1191